### DISCUSSION TITLE
2018/04/17/hosting-an-asp-net-core-2-application-on-a-raspberry-pi/

### DISCUSSION THREAD

**Discussion by giscus** [thomaslevesque/blog#108] [Blog comments]
# 2018/04/17/hosting-an-asp-net-core-2-application-on-a-raspberry-pi/

https://thomaslevesque.com/2018/04/17/hosting-an-asp-net-core-2-application-on-a-raspberry-pi/

**Comment by bymulker** [comment]
I just want to say that this helped me a lot! This post clearly explains how to setup nginx as a host and run it as a service. Thanks!

**Reply by thomaslevesque** [reply]
Glad it helped, even though it's quite out of date!

**Comment by bymulker** [comment]
Actually, i havent tried exposing specific path before. Today i did it, but  the site simply refused loading (asked to reload etc.) I turned back to root path such as location /MyWebApp/ -> location / , but now (although i reloaded) nginx, the root address is still   location /MyWebApp/ 😊 I have a simple authorization page as the start. After authorization the pages are ok such as http://192.168.0.../accounts etc. 

I just cant understand why nginx sticks to /MyWebApp/ although i turned back to location / in the configuration file.

Could you please give a tip?

Burak


Windows 10 için Posta ile gönderildi

Kimden: Thomas Levesque
Gönderilme: 10 Haziran 2024 Pazartesi 20:23
Kime: thomaslevesque/blog
Bilgi: bymulker; Comment
Konu: Re: [thomaslevesque/blog]2018/04/17/hosting-an-asp-net-core-2-application-on-a-raspberry-pi/(Discussion #108)

Glad it helped, even though it's quite out of date!

**Reply by thomaslevesque** [reply]
Hey,
Sorry, I wrote this a long time ago, and I don't really know nginx very well... Maybe try rebooting? Or look for other places where there might still be a reference to "/MyWebApp/"?
Also, all of this was done with ASP.NET Core 2, which is pretty old now. Some things might need to be done differently on latest .NET.
